crates/containerd-shim-wasm/src/sandbox/cli.rs

```diff
@@ -55,12 +60,13 @@ where
 
         std::process::exit(0);
     }
+    let shim_version = shim_version.unwrap_or("v1");
```

Comment:
```suggestion
    let shim_version = shim_version.into().unwrap_or("v1");
```

Same with `revision`.